src/drive/components/AppRoute.jsx

```diff
@@ -12,15 +14,23 @@ import {
 import { Container as Trash } from '../ducks/trash'
 
 const AppRoute = (
-  <Route component={Layout}>
-    <Redirect from="/files/:folderId" to="/folder/:folderId" />
-    <Route component={FileExplorer}>
-      <Redirect from="/" to="folder" />
-      <Route path="folder(/:folderId)" component={Folder} />
-      <Route path="recent" component={Recent} />
-      <Route path="trash(/:folderId)" component={Trash} />
+  <Route>
+    <Route component={Layout}>
+      <Redirect from="/files/:folderId" to="/folder/:folderId" />
+      <Route component={FileExplorer}>
+        <Redirect from="/" to="folder" />
+        <Route path="folder(/:folderId)" component={Folder} />
+        <Route path="recent" component={Recent} />
+        <Route path="trash(/:folderId)" component={Trash} />
+      </Route>
+      {__TARGET__ === 'mobile' && (
```

Comment:
The main router now directly includes the mobile views. Although I think this could be changed as we'll see later.

Reply:
This will cause a merge conflict with #534